README.md

```diff
@@ -1 +1,15 @@
-If you would like to contribute simulations or experiments, please follow [these instructions](https://nmrlipids.github.io/FAIRMD_lipids/stable/dbcontribute.html).
+# FAIRMD BilayerData
+FAIRMD BilayerData is an on-github database of lipid bilayer structures operated by [FAIRMD Lipids](https://github.com/NMRlipids/FAIRMD_lipids) project. The database contains information about:
+- experiments, namely form factor data from Small-Angle X-ray Scattering (SAXS) experiments and order parameter data from <sup>1</sup>H-<sup>13</sup>C dipolar splitting or from <sup>2</sup>H quadruple splitting solid-state NMR (ssNMR) experiments
+- MD simulations, run by different engines using various force fields
+- molecules with their metadata
+
+The database doesn't store raw data; we refer to public archives like Zenodo, where complete trajectories must be stored. The database structure is described in detail in [FAIRMD Lipids documentation](https://nmrlipids.github.io/FAIRMD_lipids/stable/dbstructure.html).
```

Comment:
we refer -> it refers

in [FAIRMD Lipids documentation] -> in the [FAIRMD Lipids documentation]